galasa-managers-parent/galasa-managers-zos-parent/dev.galasa.zos.manager.ivt/src/main/java/dev/galasa/zos/manager/ivt/ZosManagerFileDatasetIVT.java

```diff
@@ -124,15 +137,15 @@ private boolean checkContentWasWritten(String pds, String member, String... cont
     //Test that an existing PDS exists
     @Test
     public void testExistingDS() throws ZosBatchException, TestBundleResourceException, IOException, ZosDatasetException {
-    	String desiredDataSetName = "CTS.CICSCOG.JCL";
+    	String desiredDataSetName = "CTS.CICSCOG.JCL" + "." + getRandomString();
```

Comment:
In this case we are testing an existing dataset that already exists - we are not creating it so don't need to add the random string

Reply:
You're right, I did add this in hastily.
As an aside, is the dataset always guaranteed to exist? Should we be creating it, to guarantee it exists, for the sake of independent testing? (I don't know what `CTS.CICSCOG.JCL` is, perhaps it really will always be available on every machine we might test against?)

Reply:
You raise a good an interesting point.  Thank you for holding me to such impeccable standards.  Can you write some JCL to create a PDS dataset with a random name and use that instead.  Remember to delete it afterward